src/importexport/musicxml/internal/export/exportmusicxml.cpp

```diff
@@ -3965,20 +3965,27 @@ static bool isNoteheadParenthesis(const Symbol* symbol)
 static void writeNotehead(XmlWriter& xml, const Note* const note)
 {
     String noteheadTagname = u"notehead";
-    noteheadTagname += color2xml(note);
+    String noteheadValue;
+    if (!color2xml(note).empty()) {
+        noteheadTagname += color2xml(note);
+        noteheadValue = "normal";
```

Comment:
```suggestion
        noteheadValue = u"normal";
```
etc

to fix the "build without Qt" failure. Because if you initialise a String by a "", the conversion goes `const char*` -> `QString` -> `String`.

Reply:
Yes, my bad, like every single time. I went for the simpler solution.